packages/config/config/devices/0x027a/zen27_2.8.json

```diff
@@ -24,13 +24,15 @@
 		"2": {
```

Comment:
Can't select the correct line, but the firmwareVersion.min should be `2.8`.

Reply:
btw, the lint is failing because of this but your branch is out of date with master, so we don't see it for some reason.

Reply:
> A few nits. Also: Do you know how the association labels look like for the ones where you removed the association config? Having something like `BASIC_SET` is not so nice.

Basically everything is "BASIC_SET". What would be the preferred language? Group 2 sends on/off commands only, group 3, on the devices that support it, sends dimming events. The labels I had were just "Basic Set" and "Switch Multilevel". Maybe just "On/Off" and "Dimmer" or "On/Off Control" and "Dimmer Control"?

> btw, the lint is failing because of this but your branch is out of date with master, so we don't see it for some reason.

Yeah, I messed up when I made the branch and am not confident enough with git to fix things. I was hoping since I was only changing files that weren't changed in master that it wouldn't be an issue. If it is, I think I can stash my changes, re-create the branch off the current master, and then create a new PR.

Reply:
> The labels I had were just "Basic Set" and "Switch Multilevel". Maybe just "On/Off" and "Dimmer" or "On/Off Control" and "Dimmer Control"?

I'm fine with all of them. @blhoward2 what is most consistent with your changes?

Reply:
> If it is, I think I can stash my changes, re-create the branch off the current master, and then create a new PR.

Replace `upstream` with name my remote has:
```
git fetch upstream
git rebase upstream/master
```
If there are conflicts, fix them and check them in, then
```
git rebase --continue
```
and repeat.

In the end if all went well:
```
git push --force
```

Reply:
> > The labels I had were just "Basic Set" and "Switch Multilevel". Maybe just "On/Off" and "Dimmer" or "On/Off Control" and "Dimmer Control"?
> 
> 
> 
> I'm fine with all of them. @blhoward2 what is most consistent with your changes?

I'd say 'Basic Set - On/Off' and 'Send Dim Level' to make it clear what they do.